src/Propel/Generator/Builder/Om/TableMapBuilder.php

```diff
@@ -34,18 +34,17 @@ public function getPackage(): string
      */
     public function getNamespace(): ?string
     {
-        if (!$namespace = parent::getNamespace()) {
+        $namespace = parent::getNamespace();
+        if (!$namespace) {
             return 'Map';
         }
 
-        if (
-            $this->getGeneratorConfig()
```

Comment:
This `$this->getGeneratorConfig()` is lost in this edit.

Reply:
On purpose
The internal method does that lookup :)